src/components/table/makeSelectable.js

```diff
@@ -326,6 +471,61 @@ function makeSelectable(BaseTable) {
             this.onSelect(selectedItems.add(data[focusedIndex]), newFocusedIndex);
         };
 
+        isContinuation = (selectedItemIndecies, sourceIndex, targetIndex) => {
```

Comment:
Nits: this name is a bit ambiguous. I wasn't sure what it meant. Could we do something like `isContiguousSelection`? Also, `selectedItemIndecies` => `selectedItemIndices`. (One does not create an application to automate publication indexing without finally learning how to spell indices 😂)